apple/metal_library.bzl

```diff
@@ -21,35 +21,36 @@ load(
 )
 load("@bazel_skylib//lib:dicts.bzl", "dicts")
 load("@bazel_skylib//lib:paths.bzl", "paths")
-load(":common.bzl", "METAL_FILE_TYPES")
+load(":common.bzl", "METAL_FILE_TYPES", "DEFAULT_MINIMUM_OS_VERSION")
+load("@build_bazel_rules_apple//apple/internal:resources.bzl", "resources")
 
-def _metal_apple_target_triple(platform_prerequisites):
+def _metal_apple_target_triple(ctx):
     """Returns a Metal target triple string for an Apple platform.
     Args:
         ctx: The compilation context.
     Returns:
         A target triple string describing the platform.
     """
-    target_os_version = platform_prerequisites.minimum_os
-
-    platform = platform_prerequisites.apple_fragment.single_arch_platform
-    if platform_string == "macos":
-        platform_string = "macosx"
-
+    platform = ctx.fragments.apple.single_arch_platform
     environment = "" if platform.is_device else "-simulator"
+    target_os_version = ctx.attr.minimum_os_version
 
     return "air64-apple-{platform}{version}{environment}".format(
         environment = environment,
-        platform = platform_string,
+        platform = ctx.attr.platform,
```

Comment:
What's the reason for this change?

Reply:
@thii  Current, `platform_string` is undefined when you build for ios and the line `platform_prerequisites = platform_support.platform_prerequisites_from_rule_ctx(ctx)` show error when run( i mention in issue #40)

Reply:
This is another bug. I'll take a look.

Reply:
https://github.com/line/rules_apple_line/pull/42